scijava/scijava-ops-api/src/main/java/org/scijava/ops/api/OpBuilder.java

```diff
@@ -191,18 +191,32 @@ private void checkRefs(int mutableIndex, String label, Object... objects) {
 	 */
 	public final class Arity0 {
 
+		/**
+		 * Match a {@link org.scijava.function.Computers} op that can use the given pre-allocated output instance.
```

Comment:
I'm not sure about linking to the `Computers` container class here. It's true that the vast majority of the time, the reason to pass an output instance is because you are going to request a computer op of some arity. But not *necessarily*. It could/should match any op with a `@Container` parameter in its functional interface, I think.

Reply:
@ctrueden the `Arity0_OV` intermediate builder returned here only has [compute() and computer()](https://github.com/scijava/incubator/blob/f77076b5423522257ec3d7cc2f2e3d57e41c38e6/scijava/scijava-ops-api/src/main/java/org/scijava/ops/api/OpBuilder.java#L433-L456) methods. @gselzer does this seem incorrect to you?

Reply:
I understand that as written, this builder path is aiming toward producing only computers, not functions nor inplaces. However, I'm wondering if it might make sense for it to produce something else, of some custom functional interface compatible with the specified constraints? We don't have a matching endpoint for this currently, but one could imagine something like `.match(Class<?> customFunctionalInterface)`. Of course, if we ever were to add that, we could change the javadoc accordingly at that time, I suppose.

Reply:
I changed it anyway since it's not performing the match at this point